outputs.tf

```diff
@@ -1,4 +1,16 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+  region     = data.aws_region.current.name
+}
```

Comment:
This isn't something to worry about now but I think there's a better place to put these data sources (and maybe the locals) than just wherever. IDK where though 🤷

Reply:
Since they are only needed by the output in this case, I think this makes sense.

Reply:
I fully expected to disagree with whatever your logic was on this, but turns out I don't. I don't think I've ever _only_ used these in outputs before.